docs/docs/03-interactivity-and-dynamic-uis.md

```diff
@@ -11,22 +11,25 @@ You've already [learned how to display data](/react/docs/displaying-data.html) w
 ## A Simple Example
 
 ```javascript
-var LikeButton = React.createClass({
-  getInitialState: function() {
-    return {liked: false};
-  },
-  handleClick: function(event) {
+class LikeButton extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      liked: false
+    }
+  }
+  handleClick = () => {
```

Comment:
You can just make this a standard function, and in the constructor say `this.handleClick = this.handleClick.bind(this);`